### PR TITLE
Added 2023 members

### PR DESCRIPTION
**Pull request by GilMM27**

### CODE REVIEW

File: src/data/members.json [hunk 1]
```diff
@@ -1259,6 +1259,201 @@
...
+            "role": "Electronics",
+            "subtitle": "",
+            "class": "2023",
+            "semesters": "8",
+            "description": "I am Angel, Mechatronics Engineering Student. I am currently involved in navigation tasks. I have been enrolled in many competitions such as RoboCup Soccer and IEEE LARC. I like to workout and play tenis on my free times.",
+            "github": "https://github.com/angelcervant1",
+            "github_user": "angelcervant1",
+            "linkedin": "https://www.linkedin.com/in/angelcervant",
+            "tags": "Electronics"
+        },
```
Comment (IvanRomero03) [inline src/data/members.json]: ...

File: src/data/members.json [hunk 2]
```diff
@@ -1259,6 +1259,186 @@
...
+        {
+            "id": "125",
+            "name": "Gilberto",
+            "lastname": "Malagamba",
+            "status": "active",
+            "role": "Software Development",
+            "subtitle": "",
+            "class": "2023",
+            "semesters": "8",
```
Comment (Oscar-gg) [inline src/data/members.json]: Fix semesters

**Review by Oscar-gg — CHANGES_REQUESTED** [review]

**Comment by IvanRomero03** [comment]
ping on this pr, @GilMM27 hows the update going?

**Review by IvanRomero03 — APPROVED** [review]
lgtm if no more members to be added

**Comment by Oscar-gg** [comment]
I think the following are pending:
- Fix image dimensions (Amin Image)
- Move inactive members to inactive section

**Review by Oscar-gg — APPROVED** [review]

**Comment by Oscar-gg** [comment]
Lgtm 🚀

**Pull request merged**